packages/component-library/src/MultiLayerMap/MultiSmallPolygonMap.js

```diff
@@ -1,12 +1,12 @@
 import React from "react";
 import { PolygonLayer } from "deck.gl";
+import shortid from "shortid";
 import { number, string, bool, func, arrayOf, shape } from "prop-types";
 
 import { createColorScale, createSizeScale } from "./createLayers";
 
 const MultiSmallPolygonMap = props => {
   const {
-    index,
```

Comment:
@mendozaline just checking that you didn't have any plans for the index prop and that key isn't referenced by something else

Reply:
Nope, these changes look good. Great job, @cosmolightfoot!

Reply:
Thanks @mendozaline!